docs/index.asciidoc

```diff
@@ -96,6 +94,8 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 [cols="<,<,<",options="header",]
 |=======================================================================
 |Setting |Input type|Required
+| <<plugins-{type}s-{plugin}-user>> |<<string,string>>|no
+| <<plugins-{type}s-{plugin}-password>> |<<password,password>>|No
```

Comment:
The hrefs here are weird now because everything doing `<<password,password>>` on this page is linking to the password option. Is that a bug in the doc generator?